src/main/java/com/google/sps/servlets/BusinessServlet.java

```diff
@@ -0,0 +1,58 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.io.FileReader;
+import com.opencsv.*;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/business")
+public class BusinessServlet extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      String bus= csvToJson("/home/skolb/software-product-sprint/team13/src/main/webapp/businesses.csv");
```

Comment:
I'll definitely look into it, the only thing I'm a little worried about is that it would change how I am able to get the data, I can no longer use double index